.travis.yml

```diff
@@ -0,0 +1,21 @@
+## -*- coding: utf-8 -*-
+# 
+# Copyright (c) 2016, Veritomyx, Inc.
+#
+# This file is part of the Python SDK for PeakInvestigator
+# (http://veritomyx.com) and is distributed under the terms
+# of the BSD 3-Clause license.
+
+sudo: false
+dist: trusty
+language: python
+python:
+  - 3.6
+before_install:
+  - pip install requests
+  - pip install paramiko
+  - pip install simplejson
```

Comment:
Can you collapse these dependencies into a [`requirements.txt`](https://pip.pypa.io/en/stable/user_guide/#requirements-files) file that can be installed with `pip`?